packages/core/src/providers/yandex.ts

```diff
@@ -44,21 +44,19 @@ export interface YandexProfile {
   /**
    * ID of the Yandex user's profile picture.
    * The profile picture with this ID can be downloaded via a link that looks like this:
-   * https://avatars.yandex.net/get-yapic/<default_avatar_id>/<size>
-   * Available sizes:
-   * islands-small: 28×28 pixels.
-   * islands-34: 34×34 pixels.
-   * islands-middle: 42×42 pixels.
-   * islands-50: 50×50 pixels.
-   * islands-retina-small: 56×56 pixels.
-   * islands-68: 68×68 pixels.
-   * islands-75: 75×75 pixels.
-   * islands-retina-middle: 84×84 pixels.
-   * islands-retina-50: 100×100 pixels.
-   * islands-200: 200×200 pixels.
-   * Example: https://avatars.yandex.net/get-yapic/31804/BYkogAC6AoB17bN1HKRFAyKiM4-1/islands-200
+   * @example "https://avatars.yandex.net/get-yapic/31804/BYkogAC6AoB17bN1HKRFAyKiM4-1/islands-200"
    */
-  default_avatar_id?: string
+  default_avatar_id?:
+    | "islands-small"
+    | "islands-34"
+    | "islands-middle"
+    | "islands-50"
+    | "islands-retina-small"
+    | "islands-68"
+    | "islands-75"
+    | "islands-retina-middle"
+    | "islands-retina-50"
+    | "islands-200"
   /**
```

Comment:
@balazsorban44  `default_avatar_id` is an id in a form of string. You changed it to be an union of sizes  which is wrong. 
Here's how avatar urls formed `https://avatars.yandex.net/get-yapic/<default_avatar_id>/<size>`. take a look at `<default_avatar_id>/<size>`. these `islands-*` all are possible sizes of avatar that can be retrieved from yandex avatar cdn

Reply:
> Thanks! Added some final touches, but looks good! Nice work!

Thank you. I am really new to all this stuff. It's very nice of you taking your time and giving me feedback and helping me through. I appreciate it :smile:

Reply:
oh, I see. will follow up in a PR. it was failing the docs build and o haven't dug deep enough. 😬